TM1py/Utils/Utils.py

```diff
@@ -74,6 +74,32 @@ def get_all_servers_from_adminhost(adminhost='localhost') -> List:
         servers.append(server)
     return servers
 
+def create_server_on_adminhost(adminhost='localhost', server_as_dict={}):
```

Comment:
default values in functions should be immutable. Like you can set it to `None` instead. 

https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/

Reply:
Update to immutable.